addons/web/static/src/main.js

```diff
@@ -1,5 +1,6 @@
 /** @odoo-module **/
 
+import "@web/owl_next_adapter";
```

Comment:
Importing modules for their side effects is weird to me. Any reason we don't make it an iife like component_extension?